src/components/BestBooks.js

```
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import React from 'react';
+import Carousel from 'react-bootstrap/Carousel';
+import NewButton from './NewButton';
+import BookModal from './BookModal';
+import Book from './Book';
+
+class BestBooks extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      books: [],
+      shouldShowModal: false
+    };
+  }
+
+  addBook = async (newBook) => {
+    const postUrl = `${process.env.REACT_APP_SERVER_URL}/books`;
+    axios.post(postUrl, {
+      title: newBook.title,
+      description: newBook.description,
+      status: newBook.status
+    })
+      .then(data => {
+        this.setState({ books: [...this.state.books, data] });
+      })
+      .catch((err) => console.error(err));
```

Typically we wouldn't want to clutter the client console, but this is fine for now. Typically on the client, we'll want to surface the error to the user so they know what is going on. I can add something in our PR for displaying the error though.